src/components/MessageInput/hooks/useAttachments.ts

```diff
@@ -89,25 +105,43 @@ export const useAttachments = <
     [maxFilesLeft, noFiles],
   );
 
-  const removeAttachment = (id: string) => {
-    dispatch({ id, type: 'removeAttachment' });
-  };
+  const removeAttachments = useCallback(
+    (ids: string[]) => {
+      if (!(ids && ids.length)) return;
+      dispatch({ ids, type: 'removeAttachments' });
+    },
+    [dispatch],
+  );
+
+  const upsertAttachments = useCallback(
+    (attachments: (Attachment<StreamChatGenerics> | LocalAttachment<StreamChatGenerics>)[]) => {
+      if (!(attachments && attachments.length)) return;
```

Comment:
```suggestion
      if (!attachments.length) return;
```
Same here I suppose. 🤔

Reply:
adjusted